src/assets/imgs/warning.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
where'd this file come from?

Reply:
Wikimedia, does it need some sort of sign-off?

Reply:
As long as UX are aware it's fine

Reply:
I will check with them for confirmation.

Reply:
Replaced the Wikimedia SVG with a PNG provided by UX.